Dockerfile

```diff
@@ -36,41 +36,8 @@ RUN apk del build-dependencies
 RUN apk del build-base zlib-dev ruby-dev readline-dev libffi-dev libxml2-dev
 
 # Set up and warm sbt
-RUN mkdir /root/scala-212
```

Comment:
This is all covered by the warm-sbt project `+run` command https://github.com/olafurpg/warm-sbt/blob/master/build.sbt